.github/workflows/release.yml

```diff
@@ -1,100 +1,33 @@
-name: create release
+name: "Release"
 
 on:
-    push:
-        tags:
-            - "v[0-9]+\\.[0-9]+\\.[0-9]+"
-            - "v[0-9]+\\.[0-9]+\\.[0-9]+-rc[0-9]+"
+  # can be used to re-release an existing tag
+  workflow_dispatch:
+
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Does this limit the version numbers to 0 through 9?

Reply:
Reverted